src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -61,16 +61,21 @@
 __all__ = ["TextGenerationPipeline"]
 
 
+# Based off of https://huggingface.co/docs/transformers/main_classes/text_generation#transformers.GenerationConfig # noqa E501
 class GenerationDefaults:
-    num_return_sequences = 1
-    max_length = 1024
-    max_new_tokens = None
-    output_scores = False
-    top_k = 0
-    top_p = 0.0
-    repetition_penalty = 0.0
+    # Parameters that control the length of the output
+    max_length = None
+    max_new_tokens = 200
+    # Parameters that control the generation strategy used
     do_sample = False
+    # Parameters for manipulation of the model output logits
     temperature = 1.0
+    top_k = 50
```

Comment:
Just a heads up, I did not look into it in much detail, but changing top_k or/and top_p from zeros to HF defaults results in pretty wonky generated sequences, that deviate massively from sequences generated by HF torch model. Not sure why, maybe we are having some bug internally with top_k/top_p.